### ISSUE TITLE
tacl import does not support directories

### ISSUE THREAD

**Issue by huguesfontenelle** [unioslo/tsd-api-client#46]
## Background

The documentation says that it it possible to upload folders.
```
$ tacl --help
--import                Path to file or folder you want to upload.
                        This needs to be the last argument you pass to tacl.
```
But in practice, this is not supported.

## Steps to reproduce

```
tacl --import /path/to/folder/
User > pXX-hugues
Password >
OTP > 123456
Input data is a directory
This method handles files only
```

## Expected behavior

Folders do upload :-)

## Version

v2.3.0

**Comment by leondutoit** [comment]
Until recently, uploading directories were not supported by the TSD API. Currently uploading directories only works if you use the data pipelines, e.g.: `tacl --import dir --pre 'archive' --post 'restore'`.

To elaborate on your expectations, if you uploaded a directory, would you expect the directory to be created at the destination too?

**Comment by huguesfontenelle** [comment]
I was trying to find workarounds to #48, but uploading big files actually works (albeit with python3.7)
Also, I'd prefer #47 instead of supporting folders.

I guess I was expecting a behavior similar to `rsync -r`, whereby the folder structure is reproduced. However, the files are copied into a `file-import` folder, and the user anyway has to move them again somewhere else for deploy, so the benefit of this feature may not be that great. I'll let you judge :-)

**Comment by haatveit** [comment]
Directory uploads are supported in the new tacl2 CLI which was released in v2.7.0 last week.
